virttest/qemu_vm.py

```diff
@@ -1055,6 +1055,14 @@ def add_memorys(devices, params):
                 backend_options["size_mem"] = "%sM" % params["mem"]
                 if params.get("vm_mem_backend"):
                     backend_options["backend_mem"] = params.get("vm_mem_backend")
+                if params.get("vm_mem_backend") == "memory-backend-file":
+                    if not params.get("vm_mem_backend_path") == "":
```

Comment:
`if not params.get("vm_mem_backend_path"):`

Reply:
updated